tokio/src/fs/file.rs

```diff
@@ -176,6 +176,12 @@ impl File {
         }
     }
 
+    #[deprecated(note = "use `tokio::io::AsyncSeekExt::seek` instead")]
```

Comment:
Inherent methods take precedence over trait methods, so the only way to call seek on a File without a deprecation warning after this change would be to use `AsyncSeekExt::seek(&mut my_file)`

Reply:
Ah yes... we can just remove the deprecation notice for now.

Reply:
I just left the fn "as is".